collection/file-managers/gather-cuteftp-information.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: gather cuteftp information
+    namespace: collection/file-managers
+    author: "@_re_fox"
+    scope: function
+    att&ck:
+      - Credential Access::Credentials from Password Stores [T1555]
+    references:
+      - https://www.globalscape.com/cuteftp
```

Comment:
This page seems to be down
> The website encountered an unexpected error. Please try again later.
Error: Class 'Drupal\field_collection\Plugin\Field\FieldType\FieldCollection' not found in Drupal\field\Entity\FieldStorageConfig->getSchema() (line 453 of core/modules/field/src/Entity/FieldStorageConfig.php).

Reply:
I think that may be a recent error on the company page.  It's listed as the primary link.  I added an additional Wikipedia link in https://github.com/fireeye/capa-rules/pull/337/commits/a3899f67dce9d41cfe736c8e993e10caa854756f